src/Roslyn.Suppressions/Roslyn.Suppressions/InProcess/RoslynSettingsFileSynchronizer.cs

```diff
@@ -130,5 +136,51 @@ public void Dispose()
         {
             suppressedIssuesMonitor.SuppressionsUpdateRequested -= SuppressedIssuesMonitor_SuppressionsUpdateRequested;
         }
+
+        // Converts SonarQube issues to SuppressedIssues that can be compared more easily with Roslyn issues
+        internal static class IssueConverter
```

Comment:
I made this a nested class since this logic is only used by this class, so it doesn't need to pollute the namespace. I didn't think it was worth introducing an interface for it.

Reply:
I think it would've been cleaner to have an interface that we can mock in `UpdateFileStorageAsync` tests and then test the conversion separately. We can leave it like that for now.